packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.pcss

```diff
@@ -13,21 +13,24 @@
   border-radius: 0px var(--atomic-border-radius) var(--atomic-border-radius) 0px;
 }
 
+.box-container .value-box {
+  @apply hover:border-primary-light rounded-none p-2 border-2 flex truncate;
```

Comment:
I'm not sure why we are using an `@apply` directive here (and in a lot of places in this file) instead of classes directly on the element ? 🤔 

Is there a reason that the "classic" way does not work ?

Reply:
I've generally seen JF recommend it, so I've been using `@apply` where possible. I think both approaches are fine.

Reply:
But why is `@apply` better then ? 🤔

Reply:
Since I'm reusing some components (e.g the facet-value-box component) that are used by other kinds facets, I use the @apply directive to override some CSS properties for the segmented facet only. If I changed the class directly on the component, it would affect the other facets as well.

Reply:
Apply is better than writing actual css (ex display: none) but the best is using the Tailwind classes directly in the JSX, of course

+1 for not using FacetValueBox, since it looks pretty different

Reply:
I'll create a new functional component !